static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -281,11 +281,11 @@ const makeEpicABTestVariant = (
                 this.countryGroups.length === 0 ||
                 userMatchesCountryGroups(this.countryGroups);
 
-            const matchesTags =
-                this.tagIds.length === 0 || pageMatchesTags(this.tagIds);
-            const matchesSections =
-                this.sections.length === 0 ||
-                pageMatchesSections(this.sections);
+            const matchesTagsOrSections =
+                (this.tagIds.length === 0 && this.sections.length === 0) ||
```

Comment:
this means needs both to take account of either

Reply:
what if you want to do one that just tags or just sections?

Reply:
This line allows for no tag/section filtering if neither is set

Reply:
ahh yes sorry I completely misread it

Reply:
🤕